config/locales/en.yml

```diff
@@ -797,6 +797,9 @@ en:
         preamble: Provide in-depth information about how the server is operated, moderated, funded.
         rules_hint: There is a dedicated area for rules that your users are expected to adhere to.
         title: About
+      allow_referrer_origin:
+        desc: Control referrer header on external links. It is recommended to disable this for small instances that may not want to expose users.
+        title: Allow referrer origin on external links
```

Comment:
How about:

> **Allow external sites to see your Mastodon server as a traffic source**
> When your users click links to external sites, their browser may send the address of your Mastodon server as the referrer. Disable this if this would uniquely identify your users, e.g. if this is a personal Mastodon server.